app/views/shared/_statements_table.html.haml

```diff
@@ -9,7 +9,7 @@
         %th= Statement.human_attribute_name(:sent_to)
         - if local_assigns[:admin_view]
           %th= Account.model_name.human
-        - else
+        - elsif cross_facility?
```

Comment:
I'm not sure these should be `elsif`s. Depending on where this partial is shared, this should probably be nested under the `if local_assigns[:admin_view]`, or be a complete independent `if`.